src/main/resources/mybatis/collections.xml

```diff
@@ -129,9 +129,14 @@
                                                 ||'/observations/statistical-time-series/'
                                                 ||time_series_unique_id
                                                    as "url"
-                                       from nwis.groundwater_daily_value gw
-                                      where gw.monitoring_location_id = ml.monitoring_location_id
-                                   ) as sts
+                                      from nwis.groundwater_daily_value gw
+                                      where
+                                          gw.monitoring_location_id = ml.monitoring_location_id
+                                          AND (
+                                              gw.approvals ?? 'Approved'
+                                              OR CURRENT_DATE - gw.time_step <= 1095
```

Comment:
This is really the key change.
NOTE:  The PostGres JSON ? Operator (Does this key or value exist in the JSON object) conflicts with the JDBC ? operator for variable replacement in prepared statements.  As a work-around, the PostGres JDBC driver supports a '??' operator which essentially escapes the '?' to pass through to the DB w/o JDBC interpreting it as a parameter.